src/main/java/org/osiam/client/AuthService.java

```diff
@@ -403,6 +438,30 @@ public Builder setClientSecret(String clientSecret) {
         }
 
         /**
+         * Set the connect timeout per connector, in milliseconds.
+         *
```

Comment:
[My comment above](https://github.com/osiam/connector4java/pull/167/files#r41042335) is better placed here, but actually it's no problem to documented here